app/src/main/java/de/tum/in/tumcampusapp/widgets/TimetableWidgetConfigureActivity.java

```diff
@@ -0,0 +1,6 @@
+package de.tum.in.tumcampusapp.widgets;
+
+import android.app.Activity;
+
+public class TimetableWidgetConfigureActivity extends Activity {
+}
```

Comment:
Are we missing something here? If thats the case you should add a `TODO` marker.